product_taxes_group/tests/test_product_taxes_group.py

```diff
@@ -0,0 +1,127 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Database Integrity module for OpenERP
+#    Copyright (C) 2014 GRAP (http://www.grap.coop)
+#    @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import osv
+from openerp.tests.common import TransactionCase
+
+
+class TestProductTaxesGroup(TransactionCase):
+    """Tests for 'DataBase Integrity' Module"""
+
+    def setUp(self):
+        super(TestProductTaxesGroup, self).setUp()
+        cr, uid = self.cr, self.uid
+        self.imd_obj = self.registry('ir.model.data')
+        self.pp_obj = self.registry('product.product')
+        self.tg_obj = self.registry('tax.group')
+        self.wctg_obj = self.registry('wizard.change.tax.group')
+        self.main_company_id = self.imd_obj.get_object_reference(
+            cr, uid, 'base', 'main_company')[1]
+        self.tg1_id = self.imd_obj.get_object_reference(
+            cr, uid, 'product_taxes_group', 'tax_group_1')[1]
+        self.tg2_id = self.imd_obj.get_object_reference(
+            cr, uid, 'product_taxes_group', 'tax_group_2')[1]
+        self.pp1_id = self.imd_obj.get_object_reference(
+            cr, uid, 'product_taxes_group', 'product_product_1')[1]
+        self.at_purchase_1_id = self.imd_obj.get_object_reference(
+            cr, uid, 'product_taxes_group', 'account_tax_purchase_1')[1]
+        self.at_sale_1_id = self.imd_obj.get_object_reference(
+            cr, uid, 'product_taxes_group', 'account_tax_sale_1')[1]
+        self.at_sale_2_id = self.imd_obj.get_object_reference(
+            cr, uid, 'product_taxes_group', 'account_tax_sale_2')[1]
+
+    # Test Section
+    def test_01_change_group(self):
+        """Test if the behaviour when when we change tax group for products."""
```

Comment:
Hi again, @legalsylvain 

You should rewrite the comments of your test. Test comments are specification. If like me your are not english native, write them first in your language first.
This is crucial for QA people who are green hopers but have to live in a red world.
They have sometime to accept or not a red package for deployement deciding only on the failed test documentation.

Regards,
Charbel